egs/wsj/s5/steps/cleanup/create_segments_from_ctm.pl

```diff
@@ -324,8 +325,8 @@ sub ProcessWav {
 
   # Save the aligned CTM if needed
   if(defined($ACT)){
-    for (my $i=0; $i<=$#aligned_ctm; $i++) {
-      print $ACT "$aligned_ctm[$i][0] $aligned_ctm[$i][1] ";
+    for (my $i = 0; $i <= $#aligned_ctm; $i++) {
+      print $ACT "$wav_id $channel_id $aligned_ctm[$i][0] $aligned_ctm[$i][1] ";
```

Comment:
The right order for the CTM should be:
`print $ACT "$wav_id $channel_id $aligned_ctm[$i][1] ";`
`print $ACT "$aligned_ctm[$i][2] $aligned_ctm[$i][0] $aligned_ctm[$i][3]\n";`
Because $aligned_ctm is [$ref_word, $start, $dur, 0].   
The last element is 0 or 1, it's not a confidence, so I don't know if it should be kept but think it doesn't hurt to have this.